test/e2e/ocp_persistent_volume.go

```diff
@@ -0,0 +1,64 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+var _ = Describe("Persistent Volume", Label(smoke), func() {
+	DescribeTable("should provision PVCs", func(ctx context.Context, pvcName string) {
+		const namespace = "default"
+		manifest := fmt.Sprintf("static_resources/pvc-%s.yaml", pvcName)
+		podName := fmt.Sprintf("bb-%s", pvcName)
+		By(fmt.Sprintf("Creating a pod with %s", pvcName))
+		pod, err := loadResourcesFromYaml(manifest)
```

Comment:
If the previous comment is useful, this block would change to
```go
f, err := staticResources.Open(manifestPath)
if err != nil {
	Expect(err).NotTo(HaveOccurred())
}
defer func() {
	err := f.Close()
	Expect(err).NotTo(HaveOccurred())
}()
pod, err := loadResourcesFromReader(f)
```
changing _manifest_ to _manifestPath_.

On top of that, I think it would be a good idea to check the error of f.Close()